bin/grant-server/main.go

```diff
@@ -237,7 +237,12 @@ func main() {
 		}
 	}
 
-	srv := http.Server{Addr: ":3333", Handler: chi.ServerBaseContext(serverCtx, r)}
+	srv := http.Server{
+		Addr:         ":3333",
+		Handler:      chi.ServerBaseContext(serverCtx, r),
+		ReadTimeout:  3 * time.Second,
```

Comment:
I was looking at some CloudWatch Metrics and it seems like this service sometimes spikes at around 5-10 seconds of response time. Should we increase this timeout to 5 seconds or keep as is?

![image](https://user-images.githubusercontent.com/5459225/80732177-91f87a80-8ad1-11ea-88ba-9d004173310f.png)

Reply:
Nevermind, based on this article this looks good 👍  https://ieftimov.com/post/make-resilient-golang-net-http-servers-using-timeouts-deadlines-context-cancellation/